superset/utils.py

```diff
@@ -513,6 +513,17 @@ def table_has_constraint(table, name, db):
     return False
 
 
+def upload_file(file):
+    """Takes in the name of the file to be uploaded and uploads that file"""
+
+    from superset import app
+    config = app.config
+    if not file or not file.filename:
+        raise Exception
+    file.save(os.path.join(config['UPLOAD_FOLDER'], file.filename))
+    assert file.filename in os.listdir(config['UPLOAD_FOLDER'])
```

Comment:
Similarly what happens if an `AssertionError` is raised?

Reply:
I removed the assertion in favor of exceptions which are handled by surfacing the exception message at the top of the form.